cpp/tests/io/parquet_experimental_reader_test.cpp

```diff
@@ -142,8 +145,9 @@ auto create_parquet_with_stats()
 TEST_F(ParquetExperimentalReaderTest, TestMetadata)
 {
   // Create a table with several row groups each with a single page.
-  auto constexpr num_concat = 1;
-  auto [_, file_buffer]     = create_parquet_with_stats<num_concat>();
+  auto constexpr num_concat         = 1;
+  auto constexpr rows_per_row_group = page_size_for_ordered_tests;
+  auto [_, file_buffer]             = create_parquet_with_stats<num_concat>();
```

Comment:
If the buffer and table are not dependent on each other and you do not need the table you can do this.
```suggestion
  auto file_buffer  = std::get<1>(create_parquet_with_stats<num_concat>());
```
If the table needs to stay in scope then what you have would be correct.

Reply:
No we don't need the table